.editorconfig

```diff
@@ -0,0 +1,22 @@
+; This file is for unifying the coding style for different editors and IDEs
+; See editorconfig.org
+
+; top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{css,html,js,json}]
+indent_size = 2
+
+[*.{scss,less}]
```

Comment:
This  can be `2` instead of `4`

Reply:
Is it correct: all files must have tabs as indent. Indent size = 2?

Reply:
Yeah, I guess you make the global size equal to `2`. I'm okay with that.